mlflow/keras.py

```diff
@@ -728,7 +728,7 @@ def _run_and_log_function(self, original, args, kwargs, unlogged_params, callbac
             early_stop_callback = _early_stop_check(tmp_list[callback_arg_index])
             tmp_list[callback_arg_index] += [__MLflowKerasCallback()]
             args = tuple(tmp_list)
-        elif "callbacks" in kwargs:
+        elif kwargs.get("callbacks"):
```

Comment:
`kwargs["callbacks"]` may still be `None`, as this is the default value for the argument. For example, `fit_generator()` calls `fit()` with `callbacks=None` in Keras >= 2.4.0. Many of our other autologging integrations address this case, but this one does not.